packages/govern-subgraph/src/utils/ids.ts

```diff
@@ -7,3 +7,11 @@ export function buildId(event: ethereum.Event): string {
 export function buildIndexedId(containerHash: string, index: number): string {
   return containerHash + index.toString()
 }
+
+export function buildEventHandlerId(
+  containerHash: string,
+  eventName: string,
+  logIndex: string
+): string {
+  return containerHash + logIndex.toString()
+}
```

Comment:
@Evalir Should we just pass the container hash + the event directly as a second parameter, to make the calls shorter?